src/event.rs

```diff
@@ -219,30 +224,39 @@ impl<N: Notify> Processor<N> {
                 processor.ctx.terminal.dirty = true;
             },
             glutin::Event::KeyboardInput(state, _code, key, mods, string) => {
+                if state == ElementState::Pressed {
```

Comment:
Can we add a comment why this check is necessary? I'm a bit confused why it's here.